src/components/editor/editor-pane/tool-bar/upload-file-picker.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Editor } from 'codemirror'
+import React from 'react'
+import { handleUpload } from '../upload-handler'
+
+export const supportedMimeTypes: string[] = [
+  'application/pdf',
```

Comment:
If we want to support pdf files, we should add special logic for them and insert `{%pdf example://uploaded-url %}` instead of the image syntax.

Reply:
Yes you're totally right with that. I'll need to think about how to do that…

Reply:
done